package.json

```diff
@@ -26,7 +75,7 @@
     "electron",
     "jitsi-meet"
   ],
-  "author": "",
+  "author": "Akshit Kr Nagpal <nagpalkrakshit@gmail.com>",
```

Comment:
please set this to "Jitsi Team <support@jitsi.org>"

Reply:
Yeah. Sure